lambdas/llama-2-13b-chat-llm/src/llm.py

```diff
@@ -0,0 +1,60 @@
+import boto3
+import json
+import os
+import io
+from typing import Dict
+
+# TEMPERATURE = os.environ.get("TEMPERATURE", 1e-10)
+# MAX_NEW_TOKENS = os.environ.get("MAX_NEW_TOKENS", 1024)  # max number of tokens to generate in the output
+
+# grab environment variables
+ENDPOINT_NAME = os.environ['ENDPOINT_NAME']
```

Comment:
Can you rename to `SAGEMAKER_ENDPOINT_NAME` so it doesn't get confused with `ENDPOINT_URL` used in the other functions.

Reply:
Addressed in the 2nd commit.